test/Swap.test.ts

```diff
@@ -75,11 +76,43 @@ describe("Swap RBTC", function () {
     await expect(swapRBTC.connect(sender).withdrawalWRBTC(halfEther, sideTokenBtc.address)).to.be.revertedWith("SwapRBTC: amount > senderBalance");
   });
 
-  it("Should Not be allowed to not owner add an address to the sidetoken", async function () {
-    await expect(swapRBTC.addSideTokenBtc(sideTokenBtc.address)).to.be.revertedWith("Ownable: caller is not the owner");
+  it("Should only owner be allowed to add a side token address", async function () {
+    await expect(swapRBTC.connect(minter).addSideTokenBtc(sideTokenBtc.address)).to.be.revertedWith("Ownable: caller is not the owner");
   });
 
-  it("Should Not be allowed to not owner remove an address to the sidetoken", async function () {
-    await expect(swapRBTC.removeSideTokenBtc(sideTokenBtc.address)).to.be.revertedWith("Ownable: caller is not the owner");
+  it("Should not allow null address", async function () {
+    await expect(swapRBTC.addSideTokenBtc("0x0000000000000000000000000000000000000000")).to.be.revertedWith("SwapRBTC: sideBTC is null");
+  });
+
+  it("Should add side token address", async function () {
+    await expect(swapRBTC.addSideTokenBtc(minter.address)).to.be.emit(swapRBTC, 'sideTokenBtcAdded');
```

Comment:
I recommend using an addres that represents a token instead of the minter address. Also we should check that the token added has 18 decimals to avoid issues